zscan-upload-plugin 1.0.2

exlegalalien opened the pull request:
![TagCheck](https://steplib-git-check.services.bitrise.io/tag?pr=4320)

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [X] __I will not move an already shared step version's tag to another commit__
- [X] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [X] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [X] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [X] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)


**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/bitrise-io/bitrise-steplib?pullRequest=4320) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/bitrise-io/bitrise-steplib?pullRequest=4320) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/bitrise-io/bitrise-steplib?pullRequest=4320) it.</sub>

exlegalalien commented:
@ofalvai Any feedback on our step?

ofalvai commented on steps/zscan-upload-plugin/1.0.1/step.yml:
```diff
@@ -0,0 +1,88 @@
+  - name: curl
+is_skippable: true
+inputs:
+- endpoint: https://ziap.zimperium.com
```
Is this supposed to be changed by the user? If so, I think the description could mention why and when this needs to be changed.

ofalvai commented on steps/zscan-upload-plugin/1.0.1/step.yml:
```diff
@@ -0,0 +1,88 @@
+    is_required: true
+    summary: Console base URL.
+    title: Endpoint
+    value_options: []
```
This can be deleted, it only needs to be present when the input is an enum with predefined values. Same for all other inputs that define this.

ofalvai commented:
Just two minor findings before we can merge this @exlegalalien

exlegalalien commented:
Thanks for the feedback, @ofalvai. I've created a new release 1.0,2 with the changes below.

> This can be deleted, it only needs to be present when the input is an enum with predefined values. Same for all other inputs that define this

Done.

> Is this supposed to be changed by the user? If so, I think the description could mention why and when this needs to be changed.

Our users would typically know what this URL refers to, but I've updated the description a bit and added a link to the README file with more details.

ofalvai commented:
Thank you @exlegalalien for the changes. I think the changes are not reflected in this PR, the newly added file is still under the `1.0.1` folder and the commit hash points to `2e72a6c38ffe1b17fbee41e2b45c1dc6e550367d`.
I think at this point it's easier to close this PR and open a new one for version 1.0.2.

godrei commented:
Hi @exlegalalien, in addition to @ofalvai 's message, here is a guide about how to fix an issue with a StepLib PR: https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/sharing-steps-with-all-bitrise-users.html#fixing-issues-in-a-steplib-pull-request

exlegalalien commented:
Thanks for the feedback, @ofalvai and @godrei.  I'll create a new PR with the latest changes.  Please review/approve when you get a chance.
